cmd/admin-subnet-health.go

```diff
@@ -18,653 +18,20 @@
-	OnUsageError: onUsageError,
-	Action:       mainSubnetHealth,
-	Before:       setGlobalsFromContext,
-	Flags:        append(adminHealthFlags, globalFlags...),
-	CustomHelpTemplate: `NAME:
-  {{.HelpName}} - {{.Usage}}
-
-USAGE:
-  {{.HelpName}} TARGET
-
-FLAGS:
-  {{range .VisibleFlags}}{{.}}
-  {{end}}
-EXAMPLES:
-  1. Upload MinIO health report for 'play' (https://play.min.io by default) to SUBNET
-     {{.Prompt}} {{.HelpName}} play
-
-  2. Upload MinIO health report for alias 'play' (https://play.min.io by default) to SUBNET proxying via https://192.168.1.3:3128
-     {{.Prompt}} {{.HelpName}} play --subnet-proxy https://192.168.1.3:3128
-
-  3. Schedule periodic upload of MinIO health report for alias 'play' (https://play.min.io by default) to SUBNET every 2 days
-     {{.Prompt}} {{.HelpName}} play --schedule 2
-
-  4. Generate MinIO health report for alias 'play' (https://play.min.io by default) save and upload to SUBNET manually
-     {{.Prompt}} {{.HelpName}} play --airgap
-`,
-}
-
-// checkAdminHealthSyntax - validate arguments passed by a user
-func checkAdminHealthSyntax(ctx *cli.Context) {
-	if len(ctx.Args()) == 0 || len(ctx.Args()) > 1 {
-		cli.ShowCommandHelpAndExit(ctx, "health", 1) // last argument is exit code
-	}
-}
-
-// compress and tar MinIO health output
-func tarGZ(healthInfo interface{}, version string, filename string, showMessages bool) error {
-	f, err := os.OpenFile(filename, os.O_CREATE|os.O_RDWR, 0o666)
-	if err != nil {
-		return err
-	}
-	defer f.Close()
-
-	gzWriter := gzip.NewWriter(f)
-	defer gzWriter.Close()
-
-	enc := gojson.NewEncoder(gzWriter)
-
-	header := struct {
-		Version string `json:"version"`
-	}{Version: version}
-
-	if err := enc.Encode(header); err != nil {
-		return err
-	}
-
-	if err := enc.Encode(healthInfo); err != nil {
-		return err
-	}
-
-	if showMessages {
-		warningMsgBoundary := "*********************************************************************************"
-		warning := warnText("                                   WARNING!!")
-		warningContents := infoText(`     ** THIS FILE MAY CONTAIN SENSITIVE INFORMATION ABOUT YOUR ENVIRONMENT **
-     ** PLEASE INSPECT CONTENTS BEFORE SHARING IT ON ANY PUBLIC FORUM **`)
-
-		warningMsgHeader := infoText(warningMsgBoundary)
-		warningMsgTrailer := infoText(warningMsgBoundary)
-		console.Printf("%s\n%s\n%s\n%s\n", warningMsgHeader, warning, warningContents, warningMsgTrailer)
-		console.Infoln("MinIO health report saved at", filename)
-	}
-
-	return nil
-}
-
-func infoText(s string) string {
-	console.SetColor("INFO", color.New(color.FgGreen, color.Bold))
-	return console.Colorize("INFO", s)
-}
-
-func greenText(s string) string {
-	console.SetColor("GREEN", color.New(color.FgGreen))
-	return console.Colorize("GREEN", s)
-}
-
-func warnText(s string) string {
-	console.SetColor("WARN", color.New(color.FgRed, color.Bold))
-	return console.Colorize("WARN", s)
+	Name:               "health",
```

Comment:
I think you can hide this omment with Hidden, can you check ?

Reply:
Done